### ISSUE TITLE
Geocode not working

### ISSUE THREAD

**Issue by mayanksuman** [geopandas/geopandas#739]
The code 
```
import geopandas as gpd
gpd.tools.geocode('Kualalampur', provider="google")
```

fails with following error.
```
AttributeError                            Traceback (most recent call last)
~/.local/lib/python3.6/site-packages/geopandas/tools/geocoding.py in _query(data, forward, provider, **kwargs)
    119     try:
--> 120         Yahoo = geopy.geocoders.YahooPlaceFinder
    121     except AttributeError:

AttributeError: module 'geopy.geocoders' has no attribute 'YahooPlaceFinder'

During handling of the above exception, another exception occurred:

AttributeError                            Traceback (most recent call last)
<ipython-input-1-59f8e1cf2065> in <module>()
      1 import geopandas as gpd
----> 2 gpd.tools.geocode('Kualalampur', provider="google")

~/.local/lib/python3.6/site-packages/geopandas/tools/geocoding.py in geocode(strings, provider, **kwargs)
     60 
     61     """
---> 62     return _query(strings, True, provider, **kwargs)
     63 
     64 

~/.local/lib/python3.6/site-packages/geopandas/tools/geocoding.py in _query(data, forward, provider, **kwargs)
    120         Yahoo = geopy.geocoders.YahooPlaceFinder
    121     except AttributeError:
--> 122         Yahoo = geopy.geocoders.Yahoo
    123 
    124     coders = {'googlev3': geopy.geocoders.GoogleV3,

AttributeError: module 'geopy.geocoders' has no attribute 'Yahoo'
```
May be the attribute `YahooPlaceFinder` of `geopy.geocoders` has been removed in recent changes.

**Comment by mayanksuman** [comment]
@jdmcbr using the master solved the problem. Thanks.

**Comment by jdmcbr** [comment]
Great, glad it worked, and further incentive to get a release out soon!

**Comment by omrakn** [comment]
How would I install a package from master. Could anyone explain it to me please?
Thanks.

**Comment by jGaboardi** [comment]
@omrakn, try this from the command line:

```
pip install git+https://github.com/geopandas/geopandas
```

I am on OSX / python 3.6 and it works for me.

**Comment by jorisvandenbossche** [comment]
@omrakn It is not needed any more to install from master. Normally, the fix should be in the last released version 0.4 that can be installed using the normal ways.
